haproxy.cfg

```diff
@@ -26,38 +26,40 @@ frontend http_front
     bind *:80
     mode http
     log global
+    log-format "%ci - - [%t] \"%r\" %ST %B \"%[capture.req.hdr(1)]\" \"%[capture.req.hdr(2)]\""
 
-    capture request header host len 31
-    log-format "%ci:%cp [%t] %ft %b/%s %TR/%Tw/%Tc/%Tr/%Tt %ST %B %tsc %{+Q}r captured_host=%[capture.req.hdr(0)]"
+    redirect scheme https code 301 if !{ ssl_fc }
 
-    # Redirect HTTP to HTTPS if requesting equitytool on 80
-    acl host_equitytool req.hdr(host) -i -m beg equitytool.austinmobility.io
-    redirect scheme https code 301 if host_equitytool
 
-    # Only go to postgrest if the path is valid AND:
-    #   - either source is 10.0.0.0/8
-    #   - or source is 162.89.100.0/24 and host header is present
+frontend https_front
+    bind *:443 ssl crt /usr/local/etc/haproxy/ssl/equitytool.austinmobility.io.pem crt /usr/local/etc/haproxy/ssl/atd-postgrest.austinmobility.io.pem alpn h2,http/1.1
+    mode http
+    option httplog
+
+    capture request header host len 256
+    capture request header referer len 512
+    capture request header user-agent len 512
+
+    log-format "%ci - - [%t] \"%r\" %ST %B \"%[capture.req.hdr(1)]\" \"%[capture.req.hdr(2)]\" backend=%s/%b"
```

Comment:
I'm adopting the standard "Apache" style log output for web requests. We don't capture it any longer than we keep the container running. Calling it the "standard format" might be a bit of stretch, but if there were a standard, this would be it.